### DISCUSSION TITLE
how to define the corresponding relationship of 24 variants in ORTools

### DISCUSSION THREAD

**Discussion by wanghaijian217** [ORTools4MTEX/ORTools#61] [Q&A] [answered]
Hi Frank and Azdiar!
Thank you very much for answering my question. I recently calculated the double shear model of KS relationship. It is found that the shear mode of 24 variants has changed, and it is carried out along different double shear directions. Therefore, the orientation relationship between variants is not consistent with that defined by Morito et al., so I want to know how to redefine variants in ORtool and then draw them.
Morito:doi:10.1016/S1359-6454(02)00577-3
![image](https://github.com/ORTools4MTEX/ORTools/assets/131143952/058249da-18c9-4872-adbf-8a66727c80dc)
Thanks!
wanghaijian

**Comment by wanghaijian217** [comment]
![image](https://github.com/ORTools4MTEX/ORTools/assets/131143952/82743c15-58bf-4569-9d13-600c94dfcc15)

**Reply by AzdiarGazder** [reply, marked as answer]
Hi @wanghaijian217,

I believe you asked this question in an [earlier discussion](https://github.com/ORTools4MTEX/ORTools/discussions/54) that was never replied to or closed.

The answer remains the same as the last 2 replies.
Please provide the script you have created.
In this way, we can analyse your outputs and find ways on how best to apply them within the ORTools framework.

PS - the place you're suggesting is not where any changes can be implemented.

Warm regards,
Azdi
